src/Codeception/Lib/Actor/Shared/Comment.php

```diff
@@ -1,45 +1,47 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\Lib\Actor\Shared;
 
+use Codeception\Scenario;
+
 trait Comment
 {
-    /**
-     * @return \Codeception\Scenario
-     */
-    abstract protected function getScenario();
+    abstract protected function getScenario(): Scenario;
 
-    public function expectTo($prediction)
+    public function expectTo(string $prediction)
     {
         return $this->comment('I expect to ' . $prediction);
     }
 
-    public function expect($prediction)
+    public function expect(string $prediction)
     {
         return $this->comment('I expect ' . $prediction);
     }
 
-    public function amGoingTo($argumentation)
+    public function amGoingTo(string $argumentation)
     {
         return $this->comment('I am going to ' . $argumentation);
     }
 
-    public function am($role)
+    public function am(string $role)
     {
         $role = trim($role);
 
-        if (stripos('aeiou', $role[0]) !== false) {
+        if (stripos('aeiou', (string) $role[0]) !== false) {
```

Comment:
Is this type casting really necesssary?

Reply:
It's not, it seems like I added this before typing the parameter and then forgot to remove it, sorry.